blog/fine-grained-news-2025-10.md

```diff
@@ -0,0 +1,72 @@
+
+- [.NET SDK 0.8.0](https://github.com/openfga/dotnet-sdk/releases/tag/v0.8.0) adds support specifying [customer headers per request](https://github.com/openfga/dotnet-sdk?tab=readme-ov-file#per-request-headers), and to specify [conflict resolution options for Write operations](https://github.com/openfga/dotnet-sdk?tab=readme-ov-file#conflict-options-for-write-operations).  It builds on the [.NET SDK 0.7.0](https://github.com/openfga/dotnet-sdk/releases/tag/v0.7.0) adds support for .NET Standard 2.0, .NET 8 and .NET 9.
+
+- [Java SDK 0.9.1](https://github.com/openfga/java-sdk/releases/tag/v0.9.1) fixes issues when overriding request headers.
+
+- [Python SDK 0.9.7](https://github.com/openfga/python-sdk/releases/tag/v0.9.7) adds support for specifying [custom headers per request](https://github.com/openfga/python-sdk?tab=readme-ov-file#custom-headers), [v0.9.6](https://github.com/openfga/python-sdk/releases/tag/v0.9.6) had also brought in some enhancements.
+
+- [CLI 0.7.5](https://github.com/openfga/cli/releases/tag/v0.7.5) simplifies retrieving all tuples and ignores duplicate writes when importing tuples from a file.
+
+## 🌟 Community Spotlight
+
+[Read.AI](https://read.ai/), one of the earliest OpenFGA adopters, [was recognized by OpenAI for processing 1 trillion tokens](https://www.linkedin.com/posts/readinc_huge-shout-out-to-this-crew-and-all-read-activity-7384229591386783744-noSI). They use OpenAI for RAG use cases powered by OpenFGA :).
+
+Congrats to [Luke Woloszyn](https://www.linkedin.com/in/lukewoloszyn/), [Andrew Powers](https://www.linkedin.com/in/andrew-powers-geo/), and [Sam Matthews](https://www.linkedin.com/in/mapsam/) for such a great achievement!
+
+## 📣 Community Updates & Events
+
+### Recent Talks & Content
+
+- [Carla Urrea](https://www.linkedin.com/in/carlastabile/) published [a LinkedIn Course about OpenFGA en Español](https://www.linkedin.com/learning/openfga-implementacion-de-fine-grained-authorization/que-es-openfga-y-por-que-usarlo) and a blog post about [Understanding ReBAC and ABAC Through OpenFGA and Cedar](https://auth0.com/blog/rebac-abac-openfga-cedar/). A repository with the examples used in the blog post is [here](https://github.com/openfga/openfga-cedar-comparison).
+ 
+- [Deepu K Sasidharan](https://www.linkedin.com/in/deepu05/) presented about [Delay the AI Overlords: How OAuth and OpenFGA Can Keep Your AI Agents from Going Rogue](https://www.youtube.com/watch?v=-V251N-pYYI) at DevOxx.
+
+### Upcoming Events
+
+- [Jose Padilla](https://www.linkedin.com/in/joseapadilla/) from Okta will be presenting at KubeCon North America, together with [Alice Gibbons](https://www.linkedin.com/in/alicejgibbons/) from Diagrid about [Design Patterns for Consistent Centralized Authorization](https://kccncna2025.sched.com/event/27Fek)
```

Comment:
Do we want to reference Dapr here too, since it is a CNFC graduated project? https://www.cncf.io/projects/dapr/